service_mesh/v2x/installing-ossm.adoc

```diff
@@ -30,4 +30,4 @@ include::modules/ossm-install-ossm-operator.adoc[leveloffset=+1]
 
 == Next steps
 
-Create a `ServiceMeshControlPlane` resource to configure the components of {SMProductShortName}. For more information, see xref:../../service_mesh/v2x/ossm-create-smcp.adoc#ossm-create-smcp[Creating the ServiceMeshControlPlane].
+The {SMProductName} Operator does not create the various {SMProductShortName} custom resource definitions (CRDs) until you deploy a {SMProductShortName} control plane. You use the `ServiceMeshControlPlane` resource to install and configure the {SMProductShortName} components. For more information, see xref:../../service_mesh/v2x/ossm-create-smcp.adoc#ossm-create-smcp[Creating the ServiceMeshControlPlane].
```

Comment:
You `can` use the ServiceMeshControlPlane resource to install and configure the {SMProductShortName} components.

Reply:
There is no other way to configure the service mesh other than using the SMCP, I don't think that adding "can" is appropriate.

Reply:
Aah ok. In that case,  just `Use the ServiceMeshControlPlane.... ` would work.